pandas/core/groupby/groupby.py

```diff
@@ -992,75 +988,65 @@ def __getattribute__(self, attr: str):
             return super().__getattribute__(attr)
 
     @final
-    def _make_wrapper(self, name: str) -> Callable:
-        assert name in self._apply_allowlist
-
+    def _op_via_apply(self, name: str, *args, **kwargs):
```

Comment:
A 1 line docstring of this function would be nice.

Reply:
Good call - thanks. Done.